packages/lodestar-beacon-state-transition/src/util/slot.ts

```diff
@@ -1,12 +1,11 @@
 import {IBeaconConfig} from "@chainsafe/lodestar-config";
-import {Number64, Slot, Epoch} from "@chainsafe/lodestar-types";
-import {intDiv} from "@chainsafe/lodestar-utils";
-import {computeStartSlotAtEpoch, computeEpochAtSlot} from ".";
+import {Epoch, Number64, Slot} from "@chainsafe/lodestar-types";
+import {computeEpochAtSlot, computeStartSlotAtEpoch} from ".";
 import {GENESIS_SLOT} from "../constants";
 
 export function getSlotsSinceGenesis(config: IBeaconConfig, genesisTime: Number64): Slot {
-  const diffInSeconds = Date.now() / 1000 - genesisTime;
-  return intDiv(diffInSeconds, config.params.SECONDS_PER_SLOT);
+  const diffInMiliSeconds = Date.now() - genesisTime * 1000;
+  return Math.round(diffInMiliSeconds / (config.params.SECONDS_PER_SLOT * 1000));
```

Comment:
I'm a bit worried about this change, but we have issue wher 1ms diff is causing us to skip slot counting because of 
https://github.com/ChainSafe/lodestar/blob/master/packages/lodestar-validator/src/api/abstract.ts#L114
where 1ms causes us to skip slot counting (condition is true and by the time we start calculating diff till next slot, it makes us wait 2 second)

Reply:
I think we should just make the validator logic smarter. Its probably more important that this function always return the correct result.

Reply:
maybe we can add a function here that returns two values, current slot and milliseconds til next slot (both using the same call to `Date.now()`)

Reply:
I think I can maybe revert back and just remove that clunky logic in validator as it was needed because we didn't have pending slot block pool